leetcode/src/69.c

```diff
@@ -0,0 +1,24 @@
+//using binary search method is one of the efficient one for this problem statement.
+int mySqrt(int x){
+int start=0;
+        int end=x;
+        long long int ans=0;
+        while(start <= end){
+               long long int mid=(start+end)/2;
+                long long int val=mid*mid;
```

Comment:
There seem to be a few indentation issues here (and in other parts of the code as well). Please fix those.

Reply:
Yeah, thanks for figuring it out.I have made the respective changes for that.